src/types/quizzes/interfaces/quizzes.interface.ts

```diff
@@ -23,3 +13,10 @@ export interface CreateQuizParams {
   description: string
   items: Question[]
 }
+
+export interface UpdateQuizParams {
+  title: Quiz['title']
+  id: Quiz['_id']
+  items?: Quiz['items']
```

Comment:
items should be required, also, it should contain a category field which is not required

Reply:
Okay, I added the category, but the functional for the category hasn't been created yet so it will be "null" for now.
About extends Omit<Quiz, 'author' | '_id'> I will do this, thanks.